canonical/ubuntu.com
# Add exam scheduling and ability to take the exam

## Done

- Add scheduling and re-scheduling for exams
- Add embedded exam environment

Fixes https://warthogs.atlassian.net/browse/CRED-232

## QA

- Visit https://ubuntu-com-11968.demos.haus/credentialing/schedule to schedule an exam
- Visit https://ubuntu-com-11968.demos.haus/credentialing/your-exams to view and take your scheduled exams
  - Verify that rescheduling an exam works
  - Verify that cancelling an exam works
  - Verify that you can take the exam at the scheduled time
